src/app.js

```diff
@@ -1 +1,40 @@
-'use strict';
+/* eslint-disable no-console */
+const params = process.argv.slice(2);
+
+const HandleParams = (paramsObj) => {
+  const operation = paramsObj[0];
+  const availableOperations = ['multiply', 'add'];
+
+
+  if (!availableOperations.includes(operation)) {
+    console.log('Operation is not supported!');
+
+    return '';
```

Comment:
why do you neet to return empty string?

Reply:
@naziamoff Linter didn't allowed me with empy return so i had to add the empty string